posthog/utils.py

```diff
@@ -581,6 +581,18 @@ def get_plugin_server_job_queues() -> Optional[List[str]]:
     return None
 
 
+def is_object_storage_available() -> bool:
+    from posthog.storage import object_storage
```

Comment:
Do we need to have this import here rather than at the top? Is there some cyclic dep going on?